test_scripts/Polices/External_UCS/External_Consent_OFF/013_ATF_Policies_External_Consent_OFF_through_ignition_cycles_disallowed_by_external_consent_entities_omitted.lua

```diff
@@ -0,0 +1,306 @@
+  self.hmiConnection:SendNotification("SDL.OnAppPermissionConsent", {
+      appID = hmi_app_id_1, source = "GUI",
+      externalConsentStatus = {{entityType = 2, entityID = 5, status = "OFF"}}
+    })
+  self.mobileSession:ExpectNotification("OnPermissionsChange"):Times(0)
+  common_functions:DelayedExp(5000)
+end
+
+--------------------------------------------------------------------------
+-- Precondition:
+-- Group001: is_consented = 0
+-- Group002: is_consented = 1
+--------------------------------------------------------------------------
+CheckGroup001IsNotConsentedAndGroup002IsNotConsented()
+
+--------------------------------------------------------------------------
+-- Precondition:
+-- Ignition OFF then ON and activate application.
+--------------------------------------------------------------------------
+IgnitionOffOnActivateApp("when_externalConsentStatus_OFF")
+
+--------------------------------------------------------------------------
+-- Main check:
+-- Group001: is_consented = 0
+-- Group002: is_consented = 1
+--------------------------------------------------------------------------
+CheckGroup001IsNotConsentedAndGroup002IsNotConsented()
+
+--------------------------------------------------------------------------
+-- Precondition:
+-- HMI sends OnAppPermissionConsent with consented function = allowed
+--------------------------------------------------------------------------
+Test["TEST_NAME_OFF_Precondition_HMI_sends_again_OnAppPermissionConsent_userConsent"] = function(self)
+  local hmi_app_id_1 = common_functions:GetHmiAppId(config.application1.registerAppInterfaceParams.appName, self)
+
+  -- hmi side: sending SDL.OnAppPermissionConsent for applications with omitted externalConsentStatus
+  self.hmiConnection:SendNotification("SDL.OnAppPermissionConsent", {
+      appID = hmi_app_id_1, source = "GUI",
+      consentedFunctions = {
+        {name = "ConsentGroup001", id = id_group_1, allowed = true},
+        {name = "ConsentGroup002", id = id_group_2, allowed = true}
+      }
+    })
+
+  self.mobileSession:ExpectNotification("OnPermissionsChange")
+  :ValidIf(function(_,data)
+      local validate_result_1 = common_functions_external_consent:ValidateHMIPermissions(data,
+        "SubscribeWayPoints", {allowed = {"BACKGROUND","FULL","LIMITED"}, userDisallowed = {}})
+      local validate_result_2 = common_functions_external_consent:ValidateHMIPermissions(data,
+        "SubscribeVehicleData", {allowed = {"BACKGROUND","FULL","LIMITED"}, userDisallowed = {}})
+
+      if(validate_result_1 == false) then commonFunctions:printError("SubscribeWayPoints doesn't have permissions for allowed = {BACKGROUND, FULL, LIMITED}") end
+      if(validate_result_2 == false) then commonFunctions:printError("SubscribeVehicleData doesn't have permissions for allowed = {BACKGROUND, FULL, LIMITED}") end
+
+      return (validate_result_1 and validate_result_2)
+    end)
+end
```

Comment:
@dboltovskyi, with this step we allow permissions for these groups. According to requirements and test description sending of externalConsentStatus_ON should not change permissions, because disallowed_by_external_consent_entities_on/off is omitted.

The same expectation have test TEST_NAME_OFF_Precondition_HMI_sends_OnAppPermissionConsent_externalConsentStatus_OFF.